@here/olp-sdk-dataservice-read/lib/client/QuadKeyPartitionsRequest.ts

```diff
@@ -81,6 +82,18 @@ export class QuadKeyPartitionsRequest {
         return this;
     }
 
+    /**
+     * A setter for the provided additional fields.
+     * 
+     * @param fields Array of strings. Array could contain next values "dataSize" | "checksum" | "compressedDataSize".
```

Comment:
Please add information about how that parameter could be useful

Reply:
Added